packages/build/src/time/report.js

```diff
@@ -1,28 +1,55 @@
-const { appendFile } = require('fs')
 const { promisify } = require('util')
 
+const StatsdClient = require('statsd-client')
```

Comment:
We are not using https://github.com/brightcove/hot-shots due to https://github.com/brightcove/hot-shots/issues/178?

Reply:
Yes :(
I am really not thrilled about adding native modules, I have had so many installation errors over the years with them... As a matter of fact, `hot-shots` fails to install on my machine. I looked it up and there's a workaround, but others might run into that problem. It also can add lots of `npm install` time, especially in CI.

I dug into the source of both libraries and wrote up the comparison [here](https://github.com/netlify/buildbot/issues/891#issuecomment-670190153).